server/views/pages/approve/InitialAppointment.test.ts

```diff
@@ -37,4 +39,29 @@ describe('View Initial appointment details - approve licence', () => {
     expect($('#initial-appointment-details > .govuk-summary-list__row').length).toBe(3)
     expect($('#initial-appointment-details > div:nth-child(3) > dt').text()).toContain('Date/time')
   })
+
+  it('should display appointment person based on the appointment person type selected', () => {
```

Comment:
This test will cover server/views/pages/selectedAppointmentPerson.njk macro functionality